_inc/client/writing/theme-enhancements.jsx

```diff
@@ -13,8 +12,8 @@ import { FormFieldset } from 'components/forms';
 import { ModuleToggle } from 'components/module-toggle';
 import { ModuleSettingsForm as moduleSettingsForm } from 'components/module-settings/module-settings-form';
 import { ModuleSettingCheckbox } from 'components/module-settings/form-components';
-import SettingsCard from 'components/settings-card';
-import SettingsGroup from 'components/settings-group';
+import { SettingsCard } from 'components/settings-card';
+import { SettingsGroup } from 'components/settings-group';
```

Comment:
Ah cool. I made that change when resolving a conflict, just to match the others. 👍